app/validators/positive_money_field_validator.rb

```diff
@@ -0,0 +1,15 @@
+class PositiveMoneyFieldValidator < ActiveModel::EachValidator
+  def validate_each(form, attribute, value)
+    if value.blank?
+      form.errors.add(attribute, form.error_msg_if_blank_or_zero)
+      return
+    end
+
+    unless value.to_s.match?(/\A-?\d+(?:\.\d+)?\z/)
+      form.errors.add(attribute, I18n.t("validators.not_a_number"))
+      return
+    end
+
+    form.errors.add(attribute, form.error_msg_if_blank_or_zero) if value.to_d <= 0
```

Comment:
wonder if there should be a default `error_msg_if_blank_or_zero` if the form does not define one. I guess there is no case like in real life use & maybe every money field that is validated for this will always have a very unique validation message.

Reply:
Good idea! I'll add "This field can not be blank. Please enter a number that is greater than 0." since its more general

Reply:
also added the default here: https://github.com/codeforamerica/vita-min/pull/5947/commits/f57400c574aa3ab3ec9c7f9ffdb07090e464b6f3